internal/config/config.go

```diff
@@ -21,17 +22,28 @@ type ConfigData struct {
 	Host        string
 	Port        int
 	AccessLevel string
+
+	// Kubernetes-specific options
+	// Map of additional tools enabled (helm, cilium)
+	AdditionalTools map[string]bool
+	// Comma-separated list of allowed Kubernetes namespaces
+	AllowNamespaces string
+	// Read-only mode for Kubernetes operations
+	ReadOnly bool
```

Comment:
I would suggest we make this ReadOnly align with
```
flag.StringVar(&cfg.AccessLevel, "access-level", "readonly", "Access level (readonly, readwrite, admin)")
```

we might need to change mcp-kubernetes as well to use access-level instead of readonly.

Reply:
makes sense, added in the new commit